content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -178,6 +178,8 @@ For more information on version skews, see:
 
 {{< tabs name="k8s_install" >}}
 {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+Please note that Kubernetes provides all official `deb` packages via
+[apt.k8s.io](https://apt.k8s.io).
```

Comment:
Possibly reword the sentence(remove "Please note"):
```
{{< note >}}
Kubernetes provides all official `deb` packages from [apt.k8s.io](https://apt.k8s.io).
{{< /note >}}
```
OR
You can download the official Kubernetes `deb` packages from [...](...).
Download the official ...

Reply:
Happy to merge without fixing that detail provided we log an issue to fix it separately.

Reply:
Thank you, I changed it as suggested.